### PR TITLE
Allow user settable weights for CIEDE2000

### PR DESCRIPTION
**Pull request by chamby**

### CODE REVIEW

File: src/Phim/Color.php [hunk 1]
```diff
@@ -1618,13 +1618,15 @@ public static function fade(ColorInterface $color, $ratio)
     }
 
     //http://www.easyrgb.com/index.php?X=DELT&H=05#text5
-    public static function getDifference(ColorInterface $color, ColorInterface $compareColor)
+    public static function getDifference(ColorInterface $color, ColorInterface $compareColor, $weights = [1,1,1])
```
Comment (TorbenKoehn) [inline src/Phim/Color.php]: Can you typehint the $weights parameter as `array`? Then I'll merge it :)

Comment (TorbenKoehn) [reply src/Phim/Color.php]: It might also be useful to make sure it's always 3 elements

`$weights = array_pad($weights, 3, 1);`

**Review by TorbenKoehn — CHANGES_REQUESTED** [review]

File: src/Phim/Color.php (hunk 1 above)
Comment (chamby) [reply src/Phim/Color.php]: Here are the changes.

**Review by TorbenKoehn — APPROVED** [review]

**Comment by TorbenKoehn** [comment]
Thanks for the contribution!

**Pull request merged**